includes/class-application.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Activitypub;
+
+use Activitypub\Collection\Followers;
+use Activitypub\Collection\Users;
+
+use function Activitypub\is_user_disabled;
+
+/**
+ * ActivityPub Application Class
+ *
+ * @author Django Doucet
+ */
+class Application {
+
+	public static function known_inboxes() {
```

Comment:
I would not add it to the application class, but to the followers collection. We should also have a function to get all inboxes regardless of the user. Maybe we can use https://github.com/Automattic/wordpress-activitypub/blob/f6b4783dedbc5ad07c42a9c49a275ecaed05a227/includes/collection/class-followers.php#L200

Reply:
yes, that looks like a great fit!